clang-tools-extra/clangd/unittests/tweaks/DefineOutlineTests.cpp

```diff
@@ -19,7 +19,7 @@ TWEAK_TEST(DefineOutline);
 
 TEST_F(DefineOutlineTest, TriggersOnFunctionDecl) {
   FileName = "Test.cpp";
-  // Not available unless in a header file.
+  // Not available for free function unless in a header file.
```

Comment:
can you add some availability tests here?

Reply:
Do you have suggestions? anon namespace in header is already covered.

Reply:
well, available for:
```
namespace {
struct Foo { void ba^r() {} };
}
```
and
```
namespace ns {
struct Foo { void ba^r() {} };
}
```
and
```
struct Foo { void ba^r() {} };
```

unavailable for:
```
namespace ns {
struct Foo { void bar(); };
void Foo::b^ar() {}
}
```